tests/apps_test.go

```diff
@@ -175,14 +175,11 @@ var _ = Describe("deis apps", func() {
 				Eventually(sess).Should(Exit(0))
 			})
 
-			// Pending: The logging subsystem is not currently installed as part of the workflow-dev
-			// chart.
-			XSpecify("that user can retrieve logs for that app", func() {
+			Specify("that user can retrieve logs for that app", func() {
 				sess, err := cmd.Start("deis logs -a %s", &user, app.Name)
```

Comment:
not too familiar with this `Start` func, so ignore me if I'm mistaken, but is the first parameter supposed to be a format string? if so, there are not enough format directives (i.e. `%s`) for the number of params given